ddp-workspace/projects/toolkit/src/lib/components/app/app.component.ts

```diff
@@ -228,4 +242,12 @@ export class AppComponent implements OnInit, AfterViewInit, OnDestroy {
             this.translate.use(locale);
         }
     }
+
+    private initAnalyticsListener(): void {
+        const events = this.analytics.analyticEvents.subscribe((event: AnalyticsEvent) => {
+            ga('send', event);
+            ga('platform.send', event);
+        });
+        this.anchor.addNew(events);
+    }
```

Comment:
I am sorry because I think my comment here was unclear/misleading. What I was hoping was that this could go in the `toolkit.module.ts` (not in in app.module.ts. Toolkit does not an app.module.ts). Intention is that the wiring up the GA would happen in in the *.module.ts instead of component.